mail_restrict_send_button/models/mail_followers.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MailFollowers(models.Model):
+    _inherit = "mail.followers"
+
+    @api.model
+    def check_user_group(self):
```

Comment:
Method name should be specific about what it does.
```suggestion
    def check_can send_message(self):
```